client/js/admin_routes.jsx

```diff
@@ -7,13 +7,13 @@ import Index                          from './components/layout/index';
 import Admin                          from './components/admin/lti_applications/index';
 import Installs                       from './components/admin/lti_installs/index';
 import NotFound                       from './components/common/not_found';
-import Instances                      from './components/admin/instances/_instances';
```

Comment:
why did you rename this from Instances to Instance, from what the file does it looks like instances makes more sense, especially since there is also a component called instance.